packs/spell-effects/spell-effect-fiery-body.json

```diff
@@ -55,9 +55,42 @@
                 "key": "DamageDice",
                 "label": "Fiery Body",
                 "predicate": [
-                    "fire"
+                    "fire",
+                    {
+                        "not": "item:fiery-body"
+                    }
```

Comment:
Is this actually needed? I don't think the effect should be getting spell damage increases, though maybe the inlines are?

Reply:
Yeah I was surprised to see but as it was before, the spell effect was buffing its own inline damage. Won't be an issue if we remove the damage rolls as you suggested so I'll get rid of this.